### PR TITLE
Vet-clinic-database: multiple-ables

### PR DESCRIPTION
**Pull request by Melaku05**
Added new table owners and species
modify  the existent animals table of vet-clinic database
Added data to the new table

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -15,3 +15,23 @@ CREATE TABLE animals (
...
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100)
+);
+
+/* Remove species column */
+ALTER TABLE animals DROP COLUMN species;
+
+/* Add species_id in animals table referenced from species table */
+ALTER TABLE animals
+ADD COLUMN species_id INT REFERENCES species (id);
```
Comment (arslanbisharat) [inline schema.sql]: - [x] Please modify the `animals` table to add column `owner_id` which is a foreign key referencing the `owners` table

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @Melaku05, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (Melaku05) [reply schema.sql]: Thank you, I did that

**Review by arslanbisharat — APPROVED** [review]
Hi @Melaku05,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**